src/Renderer/LayeredMaterial.js

```diff
@@ -8,17 +8,18 @@
 import * as THREE from 'three';
 import TileVS from './Shader/TileVS.glsl';
 import TileFS from './Shader/TileFS.glsl';
-import pitUV from './Shader/Chunk/pitUV.glsl';
-import PrecisionQualifier from './Shader/Chunk/PrecisionQualifier.glsl';
+import ShaderUtils from './Shader/ShaderUtils';
+import ShaderChunk from './Shader/ShaderChunk';
 import Capabilities from '../Core/System/Capabilities';
 import { l_COLOR, l_ELEVATION, EMPTY_TEXTURE_ZOOM } from './LayeredMaterialConstants';
 
+ShaderChunk.install(THREE.ShaderChunk, 'itowns.');
```

Comment:
this and all the includes could be change to `'itowns/'` once we THREE r95+ is supported